UnitsNet.Serialization.JsonNet.CompatibilityTests/UnitsNet.Serialization.JsonNet.CompatibilityTests.csproj

```diff
@@ -12,20 +12,16 @@
     <None Remove="obj\**" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Compile Include="..\UnitsNet.Serialization.JsonNet.Tests\UnitsNetJsonConverterTests.cs" Link="UnitsNetJsonConverterTests.cs" />
-  </ItemGroup>
-
   <ItemGroup>
     <!--Get the latest released version of UnitsNet.Serialization.JsonNet in Nuget-->
-    <PackageReference Include="UnitsNet.Serialization.JsonNet" Version="*" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
 		<DotNetCliToolReference Include="dotnet-xunit" Version="2.3.0-beta2-build3683" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\UnitsNet.Serialization.JsonNet\UnitsNet.Serialization.JsonNet.csproj" />
```

Comment:
With this change, you are only verifying that the current JSON code can serialize/deserialize the current quantity code.
We also need to ensure that some arbitrary old JSON dll can serialize/deserialize new quantity code. The purpose of this CompatibilityTests project was to link in different versions of dependencies than the normal JSON tests project.

With old I mean some version deployed prior to v4 branch.
With current I mean whatever code is local starting from v4 branch and master branch when v4 is merged.

In gitter I suggested some tests like this:
1. A test that uses v3 dll of JSON serializer to serialize "current" quantities and can deserialize again
1. A test that uses "current" JSON code to serialize "current" quantities and can deserialize again
1. A test that deserializes a "v3" JSON string into "current" quantities using "current" JSON code